### DISCUSSION TITLE
can't download MindTheGapps

### DISCUSSION THREAD

**Discussion by Plaguewave7** [LSPosed/MagiskOnWSALocal#688] [Q&A] [answered]
I keep getting an error trying to download MindTheGapps while installing MagiskOnWSALocal, it keeps erroring on me, everything else downloaded, but that particular component that is important here, is there any way to get it to download?

![image](https://github.com/LSPosed/MagiskOnWSALocal/assets/71691723/2b619334-a475-44dc-8791-36213ca3193e)

**Comment by Doddddd** [comment]
https://github.com/Howard20181/MindTheGappsBuilder/releases

**Reply by Plaguewave7** [reply]
seems that there isn't a zip for X86 64, just ARM and I am not running an ARM chipset.

**Reply by Doddddd** [reply]
<img width="202" alt="image" src="https://github.com/LSPosed/MagiskOnWSALocal/assets/88972478/ebba4830-5b16-45b8-8099-714e004e93fc">

**Reply by Plaguewave7** [reply]
alright, I got it downloaded, how do I get it to install? since there was no instructions on installing it in the provided link.

**Reply by Doddddd** [reply]
Rename it and move it to the MagiskOnWSALocal/download directory, according to your screenshot.

**Reply by Doddddd** [reply]
Then run the build script.

**Reply by Plaguewave7** [reply]
what do I rename it to? and what is the command to run the script? I am still fairly new to compiling mindthegapps from the builder as I had never used this script before, and there was no provided instructions on how to run the script.

**Reply by Plaguewave7** [reply, marked as answer]
ah there we go, got it downloading in the install script for magiskonwsalocal, just had to delete the part files since I got apt-fast installed and there was some leftover aria2.part files for it that might have been causing the problem, I did rename the file after looking at my screencap and realized I just needed to change the file name to what it says it is in the screencap, remove the leftover files that were left over by the failed download attempt, then run the installer again.
